automations/js/src/project_automation/issues.mjs

```diff
@@ -7,85 +7,113 @@ import { getBoard } from '../utils/projects.mjs'
  * @param issue {import('@octokit/rest')}
  * @param board {import('../utils/projects.mjs').Project}
  * @param card {import('../utils/projects.mjs').Card}
+ * @param core {import('@actions/core')} for logging
  */
-async function syncPriority(issue, board, card) {
+async function syncPriority(issue, board, card, core) {
```

Comment:
Purely by convention, we are placing `octokit`/`github`/`core` as the first arguments in a function.